src/user/announcements.rst

```diff
@@ -8,6 +8,16 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+2022-08-24: Dropping Python 3.7
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-Forge has been providing support for Python 3.7 for 4 years now.
+Increasingly projects are moving off it (particularly in the PyData community).
+With Python 3.11's release coming around the corner (October 3rd), conda-forge
+plans to drop Python 3.7 support when Python 3.11 comes out. This will lighten
+the load on conda-forge infrastructure and make room for the new versions the
+community would like to support.
```

Comment:
```suggestion
community would like to support.

More details can be found in issue [conda-forge-pinning-feedstock#2623]( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/2623 ). Feedback is welcome there.
```

Reply:
Pushed a few more tweaks since I forgot this was reStructuredText 🤦‍♂️. Renders correctly with GitHub now.